container-toolkit/install-guide.md

```diff
@@ -74,7 +74,7 @@ Refer to the NVIDIA [Official Drivers](https://www.nvidia.com/Download/index.asp
    $ sudo yum install -y nvidia-container-toolkit
    ```
 
-### Installing with Zypper
+### OpenSUSE/SLE:
```

Comment:
It's a little strange that this is also RPM-based but has its own section. Should we call that out above or have subsections for the two package manager types?

Reply:
Yes you have a strong point here. Will think about that.

Reply:
Happy with the current state: https://nvidia.github.io/cloud-native-docs/review/pr-158/container-toolkit/latest/install-guide.html.